ts/features/bonus/bonusVacanze/screens/ActiveBonusScreen.tsx

```diff
@@ -716,4 +717,4 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
 export default connect(
   mapStateToProps,
   mapDispatchToProps
-)(withLightModalContext(ActiveBonusScreen));
+)(withBottomSheetContext(withLightModalContext(ActiveBonusScreen)));
```

Comment:
@fabriziofff nice suggestion! I noticed the `useContext` hook, but actually didn't focused on how to use it.

I'm not actually sure if using 2 different approaches may result too chaotic. We may use at the same time `useContext` hook for `LightModal` too, but this will create a misalignment with the rest of the codebase.

I, anyway, prefer the usage of the hook.